tabs/src/components/sample/style/TableOfContents.css

```diff
@@ -1,6 +1,6 @@
 nav {
     width: 250px;
-    min-width: 250px;
+    min-width: 300px;
```

Comment:
width: 250px and min-width: 300px doesn't make sense. Also is it possible to make it responsive/dynamic (like using fit-content is possible)